docs/source/docs/level-up/intermediate_skills/08_data_validate.rst

```diff
@@ -0,0 +1,70 @@
+=============
+Level 8: Dataset Validation
+=============
```

Comment:
different upper/lower lines with the title cause the build warning. it's better to have same length with title.

Reply:
Thank you for pointing out this. I updated.